### DISCUSSION TITLE
feature needed -  Can a "select all" option be added please?

### DISCUSSION THREAD

**Discussion by silpol** [OpenTTD/OpenTTD#10737] [Q&A] [answered]
I got on Facebook comment from current user:

> Can I make a suggestion for future mods. I've just downloaded the game again, and having to select all the extras I want to download is a right pain. Can a "select all" option be added please?

In my understanding, she speaks about full re-install shall pull/download not just game but extras. Maybe this feature exists? then I could advice her how to do it. Otherwise, it could be cool to add it into backlog and see people voting for it.

**Comment by rubidium42** [comment, marked as answer]
The functionality got removed (#8363) because it was simply too expensive (in actual real world money) to maintain.

Adding it would (again) increase the bandwidth usage by about 300-400% for the perceived convenience of a few, and making it easily available for all would even make it worse. After all, how likely is it that you are going to use all base graphics sets. You will likely use OpenGFX (~3 MB) and not the others (~600 MB combined).

Though, if you still have an old savegame and you got missing NewGRFs then when trying to load that savegame it has the option to download the missing ones (if they are on the server). But downloading several hundreds of NewGRFs is just annoying for you as it increases the starting time significantly and for us, as it costs way more in bandwidth.

**Reply by TrueBrain** [reply]
To add to that, for context: there are thousands of downloadable content available on our content service. There really is no need for a player to have all of them :) Check for example https://bananas.openttd.org/package/newgrf to get an understanding how massive the amount of available content is.

**Comment by silpol** [comment]
thank you, this is great feedback!
